### PR TITLE
Cache Font

### PR DESCRIPTION
**Pull request by vbjay**
# Font Cache
Don't create new Font objects on each Paint.

## Summary

Show the right way to create the Font object and cache it.

## Details

Created a cached Font object that is disposed when the Form is disposed.  This prevents fonts from being created on every Paint.
## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

**Review by rpetrusha — APPROVED** [review]
Thanks for improving this example, @vbjay!

**Comment by vbjay** [comment]
Is there something I need to do to get https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.paint to update?

**Comment by svick** [comment]
@vbjay No, just wait until the `master` branch is merged into the `live` branch. That generally happens every few days.

**Pull request merged**
